html/host.html

```diff
@@ -6,63 +6,105 @@
-        <select id="library-model" name="libraryModel" type="chooser">
-          <option>[5] Example</option>
-          <option>[5] Template</option>
-          <option>[4] Client Perspective Example</option>
-          <option selected>[4] Disease</option>
-          <option>[4] Function</option>
-          <option>[4] Investments</option>
-          <option>[4] Predator Prey Game</option>
-          <option>[4] Root Beer Game</option>
-          <option>[3] Bug Hunters Camouflage</option>
-          <option>[3] Disease With Android Avoidance</option>
-          <option>[3] Minority Game</option>
-          <option>[3] Tragedy of the Commons</option>
-          <option>[3] Walking</option>
-          <option>[2] Disease Doctors</option>
-          <option>[2] Memory</option>
-          <option>[2] Oil Cartel Alternate</option>
-          <option>[2] Public Good</option>
-          <option>[1] BeeSmart</option>
-          <option>[1] Bug Hunters Adaptations</option>
-          <option>[1] Bug Hunters Competition</option>
-          <option>[1] Dice Stalagmite</option>
-          <option>[1] Fish Spotters</option>
-          <option>[1] Gridlock</option>
-          <option>[1] Guppy Spots</option>
-          <option>[1] PANDA BEAR</option>
-          <option>[1] Polling</option>
-          <option>[1] Polling Advanced</option>
-          <option>[1] Prisoners Dilemma</option>
-
-          <option>[a] Oil Cartel</option> <!-- requires narrowcast plots -->
-
-          <option>[0] Restaurants</option> <!-- makes no distinction between AI restaurants and player-owned ones -->
-
-          <option>[0] Bird Breeders</option> <!-- requires spectator clients -->
-          <option>[0] Gridlock Alternate</option> <!-- requires spectator clients -->
-
-          <option>[0] Client Overrides Example</option> <!-- requires view overrides -->
-          <option>[0] Critter Designers</option> <!-- requires view overrides -->
-          <option>[0] Current in a Wire</option> <!-- requires view overrides -->
-          <option>[0] MTG 1 Equal Opportunities</option> <!-- requires view overrides -->
-          <option>[0] MTG 2 Random Assignment</option> <!-- requires view overrides -->
-          <option>[0] MTG 3 Feedback Loop</option> <!-- requires view overrides -->
-          <option>[0] Sampler</option> <!-- requires view overrides -->
-        </select>
-        <label>
-          Session Name:
-          <input name="sessionName" type="text" pattern="\w.*" required>
-        </label>
-        <label>
-          Password (optional):
-          <input name="password" type="password">
-        </label>
-        <input type="submit" value="Launch">
+
+        <div class="activity-selection">
+
+          <div class="activity-description">
+            <div class="library-model-container" class="library-model-container">
+              <select class="library-model" id="library-model" name="libraryModel" type="chooser">
```

Comment:
When I click this `select` in Chromium, the list of choices does a little flicker as it appears.  Do you know what's causing this/how we can make it appear without the flicker?

Reply:
Turns out that it did this before, too.  Seems platform-dependent.

Reply:
Discussed this together & this was originally there as well, so resolving this comment...